File: php/contador while.php
<?php
$contador = 5;
while ($contador < 20) {
    $contador++;
    echo "Iteración: " . $contador . "\n";
}

while (true) {
    echo "Introduzca un valor mayor a 10: ";
    $a = intval(fgets(STDIN)); // entrada desde la consola
    if ($a > 10) {
        echo "Es correcto\n";
        break; // Salir si la condición se cumple
    } else {
        echo "Es incorrecto, pruebe de nuevo\n";
    }
}
?>

File: python/contador while.py
contador = 0
while contador <30:
        contador += 1
        print("iteracion", contador)

while True:
    print("introduzca un valor mayor a 10")
    a = int(input())
    if a > 10:
        print("Es correcto")
    else:
        print("Es incorrecto, pruebe de nuevo") 
        break